framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -165,6 +165,11 @@
     /** An empty store. */
     private Store emptyStore;
 
+    protected final IdentityHashMap<
```

Comment:
Provide some documentation and explain what is different to the cache in AnalysisResult.